code/nebula/neb.cpp

```diff
@@ -419,6 +419,18 @@ void neb2_post_level_init()
 		}
 	}
 
+	float height = Neb2_detail[MAX_DETAIL_LEVEL-1].break_y;
+	for (auto poof : Neb2_poofs) {
+		int bm_height;
+		bm_get_info(poof, nullptr, &bm_height);
+		height = MAX(height, static_cast<float>(bm_height * 2));
```

Comment:
I'm not sure I understand the connection to the bitmap's size? Since regardless of the resolution of the image it'll be displayed at whatever size the poof should be anyway. Should we not be basing break_y/x on the *screen's* size? Since 'the number of pixels offscreen' is going to count a lot less nowadays than it used to.

Reply:
That's a good point.